README.md

```diff
@@ -179,18 +180,25 @@ terrahub configure -i google_function -c component.template.output -D -y
 terrahub configure -i google_function -c component.template.output.id.value='${google_cloudfunctions_function.google_function.id}'
 terrahub configure -i google_function -c component.template.output.trigger_url.value='${google_cloudfunctions_function.google_function.https_trigger_url}'
 terrahub configure -i google_function -c build.env.variables.THUB_FUNCTION_ZIP='demo.zip'
-terrahub configure -i google_function -c build.env.variables.THUB_BUILD_PATH='..'
 terrahub configure -i google_function -c build.env.variables.THUB_FUNCTION_TXT='demo.txt'
-terrahub configure -i google_function -c build.env.variables.THUB_BUILD_OK='true'
+terrahub configure -i google_function -c build.env.variables.COMPONENT_NAME='google_function'
+terrahub configure -i google_function -c build.env.variables.OBJECT_NAME='google_storage_object'
+terrahub configure -i google_function -c build.env.variables.THUB_BUILD_PATH='..'
+terrahub configure -i google_function -c build.env.variables.THUB_BUILD_OK='false'
+terrahub configure -i google_function -c build.env.variables.THUB_BUCKET_PATH='gs://google_storage_99999999'
```

Comment:
let's use `$GOOGLE_STORAGE_BUCKET` as variable instead of hardcoding the value `google_storage_9999999`